tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1793,6 +1794,142 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 
 	})
 
+	/*
+		Verify triggering offline volume expansion and delete PVC should not keep any entry in CNSvolume
+
+		1. Create StorageClass with allowVolumeExpansion set to true.
+		2. Create PVC which uses the StorageClass created in step 1.
+		3. Wait for PV to be provisioned.
+		4. Wait for PVC's status to become Bound and note down the size
+		5. Verify there should not be any PVC entry in CNS
+	*/
+	ginkgo.It("[csi-supervisor] Verify offline volume expansion when PVC is deleted", func() {
+		ginkgo.By("Invoking Test for Volume Expansion")
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		var storageclass *storagev1.StorageClass
+		var pvclaim *v1.PersistentVolumeClaim
+		var pv *v1.PersistentVolume
+		var volHandle string
+		var err error
+
+		ginkgo.By("Create StorageClass with allowVolumeExpansion set to true, Create PVC")
+		sharedVSANDatastoreURL := GetAndExpectStringEnvVar(envSharedDatastoreURL)
+		volHandle, pvclaim, pv, storageclass = createSCwithVolumeExpansionTrueAndDynamicPVC(f, client, sharedVSANDatastoreURL, storagePolicyName, namespace)
```

Comment:
I would suggest to use a thick provisioning policy for this test

Reply:
Done, updated the test